README.md

```diff
@@ -107,20 +106,12 @@ MSVC builds of Rust additionally require an installation of Visual Studio 2013
 (or later) so `rustc` can use its linker. Make sure to check the “C++ tools”
 option.
 
-With these dependencies installed, you can build the compiler in a `cmd.exe`
-shell with:
+With these dependencies installed, you can build the compiler with:
 
 ```sh
 > python x.py build
 ```
 
-If you're running inside of an msys shell, however, you can run:
-
-```sh
-$ ./configure --build=x86_64-pc-windows-msvc
```

Comment:
Specifying `x86_64-pc-windows-msvc` is necessary, otherwise `x86_64-pc-windows-gnu` will be built.